docs/decision-records/0003-client-packages-architecture.md

```diff
@@ -0,0 +1,72 @@
+# Architecture of the client packages to support multiple themes
+
+## Status
+
+Accepted on 14/02/2025. Present at the meeting: Samuel Gomez, Johnathan Meunier,
+Martin Stievenart, Guillaume Kesterman, Paul Plancq, Jean-Lou Piermé.
+
+## Context
+
+Our design system needs to support 2 themes for the client oriented
+applications: Look and feel, and Apollo. The proposal made by Samuel Gomez is to
+create a new package for each theme. The architecture looks like this
+
+```
+client
+├───look-and-feel
+│   ├───css
+│   │   └───src
+│   │       └───Button
+│   │              Button.scss
+│   └───react
+│       └───src
+│           └───Button
+│                  Button.tsx
+├───apollo
+│   ├───css
+│   │   └───src
+│   │       └───Button
+│   │              Button.scss
+│   └───react
+│       └───src
+│           └───Button
+│                  Button.tsx
+└───common
+        └───src
+            └───Button
+                    Button.scss
+                    Button.tsx
+```
+
+In the common folder, code that can be shared between the two themes will be
+stored. For the necessary adjustments, each theme can override code. Ideally,
+the in the theme source files, we would only re-export the common component.
```

Comment:
`the in the theme source files`
je sais pas trop ce que tu as voulu dire là

Reply:
ouais, la fatigue sorry. 
Y avait un `the` en trop